style.css

```diff
@@ -0,0 +1,35 @@
+body {
+    margin: 0;
```

Comment:
Please add values to the following or delete them if you aren't going to give them a value to avoid linter erros.